static/src/javascripts/projects/commercial/modules/cmp/ccpa-cmp.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import { isInUsa } from 'common/modules/commercial/geo-utils';
+import config from 'lib/config';
+
+let ccpaApplicable;
+
+export const isCcpaApplicable = (): boolean => {
+    if (typeof ccpaApplicable === 'undefined') {
+        ccpaApplicable = isInUsa() && config.get('switches.ccpaCmpUi', true);
```

Comment:
if we test for switch _or_ ` config.get('tests.ccpaCmpVariant') === 'variant'`, then this can be merged without affecting the test. switching it on will still show to everyone in US, not just those in the test.